lib/index.js

```diff
@@ -0,0 +1,11 @@
+
+const Util = require('util');
+
+const Config = require('../config');
+const MoviesServer = require('./server');
+
+MoviesServer.start(() => {
+	Util.log(`MoviesServer started on port:${Config.PORT}`);
```

Comment:
Since the rest of your repo uses spaces, it's better to keep it consistent and change tabs to 2 spaces.

There's usually a setting in your text editor to ensure that all tabs are automatically converted.